src/layout/_settings.scss

```
@@ -23,4 +23,38 @@
     width: 100%;
     padding: 0 16px;
   }
-}
\ No newline at end of file
+}
+
+.ncgr-settings__svg-badge {
+  width: auto;
+  height: 60px;
+}
+
+.ncgr-settings__tfa-qr-code {
+  background-color: var(--color-white);
+  display: inline-block;
+  padding: 4px;
+}
+
+.ncgr-settings__tfa-secret-key {
+  display: flex;
+  border-radius: 8px;
+  border: 2px solid rgba(var(--color-textfield-separator), 0.15);
+}
+
+.ncgr-settings__tfa-recovery-code-card {
+  width: 60%;
+  margin: auto;
+  @include mediaquery(tablet) {
+    width: 100%;
+  }
+}
+
+.ncgr-settings__tfa-recovery-code-flexbox {
+  display: flex;
+  justify-content: center;
+}
+
+.ncgr-settings__tfa-copy-button {
+  display: block
```

`;`が抜けてる〜
1個だとなくてもいけるけど後々事故るのでつけるようにしましょう 👍 

```suggestion
  display: block;
```

---

あぶない！今までLocalで見た目だけ確認してました🙇‍♀️今後コードもちゃんと見返します！！